recipe/LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
```

Comment:
It looks like a re-rendering occurred in the `recipe` directory and added this stuff there, which shouldn't be there. Could you please clean out the `recipe` directory of everything except `meta.yaml` and `conda_build_config.yaml`?

Reply:
Done